recipes/ixwebsocket/all/conanfile.py

```diff
@@ -119,6 +119,10 @@ def _patch_sources(self):
             replace_in_file(self, cmakelists,
                                   "ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}",
                                   "ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR} LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR} RUNTIME DESTINATION bin")
+        # INFO: IXWebSocketHttpHeaders.cpp consumes WSAEINVAL but there is no winsock2.h included
+        replace_in_file(self, os.path.join(self.source_folder, "ixwebsocket", "IXWebSocketHttpHeaders.cpp"), '#include "IXSocket.h"', '#include "IXNetSystem.h"\n#include "IXSocket.h"')
+        # INFO: IXHttpClient.cpp consumes WSAEINVAL but there is no winsock2.h included
+         replace_in_file(self, os.path.join(self.source_folder, "ixwebsocket", "IXHttpClient.h"), '#include "IXSocket.h"', '#include "IXNetSystem.h"\n#include "IXSocket.h"')
```

Comment:
```suggestion
        # INFO: IXWebSocketHttpHeaders.cpp consumes WSAEINVAL but there is no winsock2.h included
        replace_in_file(self, os.path.join(self.source_folder, "ixwebsocket", "IXWebSocketHttpHeaders.cpp"), '#include "IXSocket.h"', '#include "IXNetSystem.h"\n#include "IXSocket.h"')
        # INFO: IXHttpClient.cpp consumes WSAEINVAL but there is no winsock2.h included
        replace_in_file(self, os.path.join(self.source_folder, "ixwebsocket", "IXHttpClient.h"), '#include "IXSocket.h"', '#include "IXNetSystem.h"\n#include "IXSocket.h"')
```
Ops, bad indentation